frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -171,6 +164,20 @@ action.GitMachete.SlideOutSelectedBranchAction.GitMacheteContextMenu.text=Slide
 action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected branch
 
 
+# Graph Table
+
+action.GitMachete.DiscoverAction.text=Discover Branch Layout
+action.GitMachete.DiscoverAction.description=Discovers the Branch Tree in Git Machete Plugin
+action.GitMachete.DiscoverAction.cant-discover-layout-error-title="Can't discover repository branch tree"
+action.GitMachete.DiscoverAction.cant-get-current-repository-error-title="Can't get current git repository"
+action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.empty-table-text="No branch has been discovered"
+action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.save-button-text=Save Discovered Layout
+action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.title=Discovered branch tree
```

Comment:
Use `branch layout` consistently so that users aren't confused by `tree vs layout`